internal/cmd/cmd.go

```diff
@@ -310,3 +372,22 @@ func (c *Cmd) Output() ([]byte, error) {
 	err := c.Run()
 	return b.Bytes(), err
 }
+
+func (c *Cmd) afterExit(ctx context.Context) (err error) {
+	for _, f := range c.afterExitFuns {
+		if ferr := f(ctx); err != nil {
+			err = ferr
```

Comment:
should we log these errors? We should consider how we can avoid losing the errors of the previous after exit functions here.